packages/app/src/components/nominationInfo/ApplicationForm.js

```diff
@@ -8,8 +8,9 @@ import { NominationsDataContext } from '../../utils/context/NominationsContext';
 import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
 import ViewCard from './ViewCard';
 import EditCard from './EditCard';
-import { isValidPhoneNumber, formatPhoneNumber} from 'react-phone-number-input'
-
+import { formatPhoneNumber } from 'react-phone-number-input'
+import "yup-phone";
+import emailExistence from "email-existence";
```

Comment:
Are we using this `email-existence` library? I don't see it called anywhere.

I'd actually be OK if we didn't go this heavy-handed when it comes to input validation. Validating that something is a legit email is an actually hard problem (sort of like validating names, you're always going to get it wrong), and so this is sort of a clever approach. But it also feels a little overzealous to do from the front end, and maybe a little abusive.

Reply:
No we are not!  I just removed and uninstalled it!